### DISCUSSION TITLE
1.7.1 Update - Project Backups!

### DISCUSSION THREAD

**Discussion by keiwando** [keiwando/toonsquid#97] [Announcements]
The 1.7.1 update is now available and contains a built-in way of creating local backups of your ToonSquid projects in order to keep your work safe.

You can read all about it here: https://toonsquid.com/handbook/projects/backup/#backup
Here are the available backup settings: https://toonsquid.com/handbook/settings/backup/

I've tried to make sure that the backup reminder never pops up while you are in the middle of drawing or editing your animation. Please let me know if you run into situations where the reminder was disturbing or if you notice any other issue.

**Comment by yaknepper** [comment]
Awesome update. Thank you.
The backup reminder pops up like every 5 minutes no matter what you do, even if you just backed up the project :-)

Cheers

**Reply by keiwando** [reply]
Is it actually 5 minutes or does it feel like 5 minutes? The default interval is 20 minutes. Have you tried changing the setting?
https://toonsquid.com/handbook/settings/backup/
